libs/features/interlinks.js

```diff
@@ -35,14 +36,16 @@ export default async function interlink(path, limit = 1000) {
   const maxLinks = (Math.floor(articleWords / 100)) - articleLinks;
   // eslint-disable-next-line no-useless-return
   if (maxLinks <= 0) return;
+  let wordBorder = '';
+  if (!exceptionLanguages.includes(language)) { wordBorder = '\\b';}
```

Comment:
nit: could save a line with a ternary. 
`const wordBorder = exceptionLanguages.includes(language) ? '' : '\\b';`

Reply:
Thanks for the suggestion. I have updated the code.